dapp/src/contracts/soroban_domain_sdk.ts

```diff
@@ -1,20 +1,23 @@
 import * as SDK from "@stellar/stellar-sdk";
 import { SorobanDomainsSDK } from "@creit.tech/sorobandomains-sdk";
+import { getNetworkConfig } from "../utils/networks";
 
-const vaultsContractId = import.meta.env.PUBLIC_SOROBAN_DOMAIN_CONTRACT_ID;
-const defaultFee = import.meta.env.PUBLIC_DEFAULT_FEE ?? "100";
-const defaultTimeout = import.meta.env.PUBLIC_DEFAULT_TIMEOUT ?? 30;
-const simulationAccount = import.meta.env.PUBLIC_TANSU_OWNER_ID;
-const networkPassphrase = import.meta.env.PUBLIC_SOROBAN_NETWORK_PASSPHRASE;
+// Create domain SDK based on current network selection
+function createDomainSDK() {
+  const config = getNetworkConfig();
+  const defaultFee = "100";
+  const defaultTimeout = 30;
+  const simulationAccount = "GCMFQP44AR32S7IRIUKNOEJW5PNWOCLRHLQWSHUCSV4QZOMUXZOVA7Q2"; // From tansu.toml
```

Comment:
Why changing from `import.meta.env.PUBLIC_TANSU_OWNER_ID`? We need this address. This is helpful for fees, etc.